massdriver.yaml

```diff
@@ -131,6 +132,27 @@ params:
                 const: p2.8xlarge
               - title: P2 General Purpose GPU 16xlarge (64 vCPUs, 732.0 GiB)
                 const: p2.16xlarge
+          advanced_configuration_enabled:
+            type: boolean
+            title: Advanced Configuration Enabled
+            default: false
+        dependencies:
+          advanced_configuration_enabled:
+            oneOf:
+              - properties:
+                  advanced_configuration_enabled:
+                    const: false
+              - properties:
+                  advanced_configuration_enabled:
+                    const: true
+                  advanced_configuration:
+                    type: object
+                    title: Advanced Configuration
+                    properties:
+                      taint:
+                        $ref: https://raw.githubusercontent.com/massdriver-cloud/artifact-definitions/main/definitions/types/k8s-node-taint.json
+                        description: "Warning: if you only have one node pool, and it has a taint, other core workloads may not be able to be scheduled and provisioning will fail. If you want to taint a node group, it's recommended that you have another, non-tainted pool for these workloads."
```

Comment:
```suggestion
                        description: "**Warning**: if you only have one node pool, and it has a taint, other core workloads may not be able to be scheduled and provisioning will fail. If you want to taint a node group, it's recommended that you have another, non-tainted pool for these workloads."
```

Super NIT, but for a warning like this, I think some markdown would help the customer with spotting the warning description before they quickly click deploy without much thought.